
> dev
> tsx watch src/index.ts

node:fs:562
  return binding.open(
                 ^

Error: ENOENT: no such file or directory, open './test/data/05-versions-space.pdf'
    at Object.openSync (node:fs:562:18)
    at Object.readFileSync (node:fs:446:35)
    at Object.<anonymous> (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/node_modules/pdf-parse/index.js:15:25)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object.transformer (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/node_modules/tsx/dist/register-D46fvsV_.cjs:3:1104)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at cjsLoader (node:internal/modules/esm/translators:262:5) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './test/data/05-versions-space.pdf'
}

Node.js v22.14.0
6:09:39 AM [tsx] change in ./src/pipeline/normalize.ts Rerunning...
node:fs:442
    return binding.readFileUtf8(path, stringToFlags(options.flag));
                   ^

Error: ENOENT: no such file or directory, open '/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/server/src/sql/schema.sql'
    at Object.readFileSync (node:fs:442:20)
    at Server.<anonymous> (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/src/index.ts:25:21)
    at Object.onceWrapper (node:events:632:28)
    at Server.emit (node:events:530:35)
    at emitListeningNT (node:net:1980:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:89:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/server/src/sql/schema.sql'
}

Node.js v22.14.0
c6:09:45 AM [tsx] change in ./src/pipeline/normalize.ts Rerunning...
node:fs:442
    return binding.readFileUtf8(path, stringToFlags(options.flag));
                   ^

Error: ENOENT: no such file or directory, open '/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/server/src/sql/schema.sql'
    at Object.readFileSync (node:fs:442:20)
    at Server.<anonymous> (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/src/index.ts:25:21)
    at Object.onceWrapper (node:events:632:28)
    at Server.emit (node:events:530:35)
    at emitListeningNT (node:net:1980:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:89:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/server/src/sql/schema.sql'
}

Node.js v22.14.0
c6:11:23 AM [tsx] change in ./src/index.ts Rerunning...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1937:16)
    at listenInCluster (node:net:1994:12)
    at Server.listen (node:net:2099:7)
    at Function.listen (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/node_modules/express/lib/application.js:635:24)
    at <anonymous> (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/src/index.ts:23:5)
    at ModuleJob.run (node:internal/modules/esm/module_job:271:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:578:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1973:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.14.0
c6:11:27 AM [tsx] change in ./src/index.ts Rerunning...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1937:16)
    at listenInCluster (node:net:1994:12)
    at Server.listen (node:net:2099:7)
    at Function.listen (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/node_modules/express/lib/application.js:635:24)
    at <anonymous> (/Users/teifionmartens/Zyloxxx/zylox-wa-demo/server/src/index.ts:23:5)
    at ModuleJob.run (node:internal/modules/esm/module_job:271:25)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:578:26)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:116:5)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1973:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.14.0
c
